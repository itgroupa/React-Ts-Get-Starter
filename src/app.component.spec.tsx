import { App } from './app.component';

import React from 'react';

import renderer from 'react-test-renderer';

describe('<App />', () => {
  test('App should create', async () => {
    const el = renderer.create(<App />)
      .toJSON();
    expect(el).toMatchSnapshot();
  });
});
