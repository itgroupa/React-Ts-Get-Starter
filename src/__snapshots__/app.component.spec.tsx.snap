// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> App should create 1`] = `
Array [
  <nav>
    <a
      href="/one"
      onClick={[Function]}
    >
      One
    </a>
    <a
      href="/two"
      onClick={[Function]}
    >
      Two
    </a>
  </nav>,
  <div>
    Loading
  </div>,
]
`;

exports[`<App /> should display a blank login form, with remember me checked by default 1`] = `
Array [
  <nav>
    <a
      href="/one"
      onClick={[Function]}
    >
      One
    </a>
    <a
      href="/two"
      onClick={[Function]}
    >
      Two
    </a>
  </nav>,
  <div>
    Loading
  </div>,
]
`;
